GetBPDPaymentInstrument/handler.ts

```diff
@@ -83,7 +86,9 @@ function toApiPaymentInstrumentDetail(
           {
             ...pi,
             cancellation: pi.cancellation?.toISOString(),
-            channel_descr: getChannel(pi.channel),
+            channel_descr: fromNullable(pi.channel)
```

Comment:
This must be removed, we are here inside `activation_periods`

Reply:
done :)